src/pages/PullRequestPage/subroute/FilesChangedTab/FilesChanged/hooks/useImpactedFilesTable.spec.js

```diff
@@ -294,4 +298,23 @@ describe('useImpactedFilesTable', () => {
       await waitFor(() => expect(flagsMock).toHaveBeenCalledWith('flag1,flag2'))
     })
   })
+
+  describe('sends components to the API', () => {
```

Comment:
Can we add a test for sending both flags and components? This is to ensure we are generating the correct path with multiple queries.

Reply:
sure thing